tests/integration/courses.test.js

```diff
@@ -160,7 +162,9 @@ describe('COURSES ROUTES - POST /courses', () => {
         type: 'invalid type',
         subProgram: subProgramsList[0]._id,
         salesRepresentative: vendorAdmin._id,
+        expectedBillsCount: 1,
```

Comment:
comme le type est invalide, il ne faut pas que tu mettes ce champ la. Ta validation joi le rend interdit car le type n'est pas INTRA `.when('type', { is: INTRA, then: Joi.required(), otherwise: Joi.forbidden() }),`

Reply:
Ha oui bien vu !